source/scenes/DayOf/scenes/Travel/index.jsx

```diff
@@ -1,7 +1,75 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 
+import travels from './travel.js';
 import './styles.scss';
 
-const Travel = () => <h1>Travel</h1>;
+type TravelPlanProps = {
+  itineraryType: string,
+  settings: object,
+};
+
+type TravelProps = {
+  university: string,
+  beforeEvent: object,
+  afterEvent: object,
+};
+
+const TravelPlan = (props: TravelPlanProps) => {
+  const { itineraryType, settings } = props;
+
+  return (
+    <Fragment>
```

Comment:
Is there a reason you're using `Fragment` here and not just returning the child `div`?

Reply:
Not really, I can change that to `div` instead of `Fragment`

Reply:
O, changed it to div and added key props for the map calls so that warnings don't show